content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -55,6 +55,24 @@ The `kubernetes.io/` and `k8s.io/` prefixes are reserved for Kubernetes core com
 
 Valid label values must be 63 characters or less and must be empty or begin and end with an alphanumeric character (`[a-z0-9A-Z]`) with dashes (`-`), underscores (`_`), dots (`.`), and alphanumerics between.
 
+For example, the sample Pod with the labels `environment: production` and `tier: frontend`:
```

Comment:
This is not a complete sentence.

Reply:
Changed statement to make it more meaningful